CHANGELOG.md

```diff
@@ -15,7 +15,8 @@ Please add to the relevant subsections under Unreleased below on every PR where
 
 ### Added
 
-- Support queue template variables in launch sweep scheduelr jobs @KyleGoyette https://github.com/wandb/wandb/pull/7787
+- Support queue template variables in launch sweep scheduler jobs @KyleGoyette https://github.com/wandb/wandb/pull/7787
+- When using wandb-core, support multipart uploads to S3 @moredatarequired https://github.com/wandb/wandb/pull/7659
```

Comment:
remember to put this under Unreleased, not 0.17.4

(we should probably have a separate CHANGELOG_unreleased.md file to avoid this problem @kptkin)

Reply:
oh and just append when we are doing a release?